app/models/user.rb

```diff
@@ -22,4 +24,8 @@ def organisations
     user = Github.new oauth_token: token, auto_pagination: true
     user.orgs.list.body.map { |org| org[:login]&.downcase }.compact
   end
+
+  def set_whitelisted_role
```

Comment:
Is this method really necessary? Seems not to be an improvement over the build-in enum syntax.